lua/apisix/plugins/zipkin.lua

```diff
@@ -0,0 +1,172 @@
+local core = require("apisix.core")
+local new_tracer = require("opentracing.tracer").new
+local zipkin_codec = require("apisix.plugins.zipkin.codec")
+local new_random_sampler = require("apisix.plugins.zipkin.random_sampler").new
+local new_reporter = require("apisix.plugins.zipkin.reporter").new
+local ngx = ngx
+local pairs = pairs
+
+local plugin_name = "zipkin"
+
+
+-- You can follow this document to write schema:
+-- https://github.com/Tencent/rapidjson/blob/master/bin/draft-04/schema
+-- rapidjson not supported `format` in draft-04 yet
+local schema = {
+    type = "object",
+    properties = {
+        endpoint = {type = "string"},
+        sample_ratio = {type = "number",
+                        default = 0.001, minimum = 0.00001, maximum = 1}
```

Comment:
The default value here may be invalid, it is best to add a use case to cover this case.

Reply:
json schema in rapidjson not support default value :(